File: learning-api/src/hello/hello.js
exports.handler = async (event) => {
  let params = event.queryStringParameters;
  let name = params && params.name ? params.name : "world";
  
  const STAGE = process.env.STAGE;

  return {
    statusCode: 200,
    headers: {
      'Content-Type': 'application/json'
    },
    body: JSON.stringify({
      message: `Hello ${name}!`,
      stage: STAGE
    })
  };
};

File: learning-api-python/src/hello/hello.py
import os
import json

def handler(event, context):
    # get name from query string parameters
    query = event.get("queryStringParameters", {})
    name = query.get("name", "World")

    # get stage from environment variables
    STAGE = os.getenv("STAGE", "dev")

    # return response
    body = {
        "message": f"Hello {name}!",
        "stage": STAGE
    }
    response = {
        "statusCode": 200,
        "headers": { "Content-Type": "application/json" },
        "body": json.dumps(body)
    }
    return response